### PR TITLE
refactor, chore(egen): Refactored code according to template guidelines, added code to delete job in cleanup section

### PR DESCRIPTION
**Pull request by sumanvita-springml**
This PR includes the following updates:

1. Adds Colab Enterprise link and other structural changes from the new template.
2. Changes REGION variable name to LOCATION.
3. Changes dag variable name to job
4. Removes os.getenv("IS_TESTING") and import os from the cleanup section.
5. Adds job.delete() to delete training job in the cleanup section. 
6. Removes code highlight when used at places other than code or variables. 
7. Adds comments for cells needed.
8. Removes the usage of future tense.
9. Replaces aip with aiplatform.
10. Contracts words like is not, do not, You are, etc.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb [hunk 1]
```diff
@@ -75,25 +77,25 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to use `AutoML` to create a tabular binary classification model from a Python script, and then learn to use `Vertex AI Online Prediction` to make online predictions with explanations. You can alternatively create and deploy models using the `gcloud` command-line tool or online using the Cloud Console.\n",
+        "In this tutorial, you learn how to use AutoML to create a tabular binary classification model from a Python script. Then, you  learn to use Vertex AI prediction service to make online prediction request with explanations. You can alternatively create and deploy models using the `gcloud` command-line tool or online using the Cloud Console.\n",
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb:80]: delete extra space and add "the" and add "a":

- Then, you  learn to use Vertex AI prediction service to make prediction request...

To:

- Then, you learn to use the Vertex AI prediction service to make a prediction request...

File: notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb [hunk 2]
```diff
@@ -75,25 +77,25 @@
...
         "\n",
-        "- `Vertex AI AutoML`\n",
-        "- `Vertex AI Prediction`\n",
-        "- `Vertex Explainable AI`\n",
-        "- `Vertex AI Model` resource\n",
-        "- `Vertex AI Endpoint` resource\n",
+        "- Vertex AI AutoML\n",
+        "- Vertex AI Prediction\n",
+        "- Vertex Explainable AI\n",
+        "- Vertex AI Model resource\n",
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb:87]: use lower case:

- Vertex AI model resource

File: notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb [hunk 3]
```diff
@@ -75,25 +77,25 @@
...
+        "- Vertex AI Endpoint resource\n",
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb:88]: use lower case:

- Vertex AI endpoint resouce

File: notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb [hunk 4]
```diff
@@ -75,25 +77,25 @@
...
         "The steps performed include:\n",
         "\n",
-        "- Create a `Vertex Dataset` resource.\n",
-        "- Train an `AutoML` tabular binary classification model.\n",
+        "- Create a Vertex AI dataset resource.\n",
+        "- Train an AutoML tabular binary classification model.\n",
         "- View the model evaluation metrics for the trained model.\n",
-        "- Create a serving `Endpoint` resource.\n",
-        "- Deploy the `Model` resource to a serving `Endpoint` resource.\n",
+        "- Create a serving Endpoint resource.\n",
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb:95]: lower case:

- endpoint resource

File: notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb [hunk 5]
```diff
@@ -75,25 +77,25 @@
...
+        "- Deploy the Model resource to a serving Endpoint resource.\n",
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb:96]: lower case:

- endpoint resource

File: notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb [hunk 6]
```diff
@@ -104,7 +106,7 @@
       "source": [
         "### Dataset\n",
         "\n",
-        "The dataset used for this tutorial is the [Iris dataset](https://www.tensorflow.org/datasets/catalog/iris) from [TensorFlow Datasets](https://www.tensorflow.org/datasets/catalog/overview). This dataset does not require any feature engineering. The version of the dataset you will use in this tutorial is stored in a public Cloud Storage bucket. The trained model predicts the type of Iris flower species from a class of three species: setosa, virginica, or versicolor."
+        "The dataset used for this tutorial is the [Iris dataset](https://www.tensorflow.org/datasets/catalog/iris) from [TensorFlow Datasets](https://www.tensorflow.org/datasets/catalog/overview). The dataset does not require any feature engineering. The version of the dataset you use in this tutorial is stored in a public Cloud Storage bucket. The trained model predicts the type of Iris flower species from a class of three species: setosa, virginica, or versicolor."
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb:109]: contraction of "does not" to "doesn't"

File: notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb [hunk 7]
```diff
@@ -414,11 +359,9 @@
       "source": [
         "#### Quick peek at your data\n",
         "\n",
-        "You will use a version of the Iris dataset that is stored in a public Cloud Storage bucket, using a CSV index file.\n",
-        "\n",
-        "Start by doing a quick peek at the data. You count the number of examples by counting the number of rows in the CSV index file  (`wc -l`) and then peek at the first few rows.\n",
+        "You can count the number of examples by counting the number of rows in the CSV index file  (`wc -l`) and then take a peek at the first few rows.\n",
         "\n",
-        "You also need for training to know the heading name of the label column, which is save as `label_column`. For this dataset, it is the last column in the CSV file."
+        "The heading name of the label column, is `label_column` which is the last column in the CSV file."
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb:364]: add comma:

- label_column`, which is the last column

File: notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb [hunk 8]
```diff
@@ -662,28 +607,28 @@
...
+        "The response from the `explain()` call is a Python dictionary with the following entries:\n",
         "\n",
         "- `ids`: The internal assigned unique identifiers for each prediction request.\n",
         "- `displayNames`: The class names for each class label.\n",
         "- `confidences`: For classification, the predicted confidence, between 0 and 1, per class label.\n",
         "- `values`: For regression, the predicted value.\n",
-        "- `deployed_model_id`: The Vertex AI identifier for the deployed `Model` resource which did the predictions.\n",
-        "- `explanations`: The feature attributions"
+        "- `deployed_model_id`: The Vertex AI identifier for the deployed model resource which generated the predictions.\n",
+        "- `explanations`: The feature attributions returned by the Explainable AI."
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_automl_tabular_classification_online_explain.ipynb:631]: remove "the"

- returned by Explainable AI

**Review by kittyabs — COMMENTED** [review]
suggested edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**